### ISSUE TITLE
Vimeo

### ISSUE THREAD

**Issue by 89z** [shroudedcode/apk-mitm#73]
Using this app:

https://apkpure.com/vimeo/com.vimeo.android.videoapp

I am able to patch using this package. If I start the app, it prompts for login. Alternatively you can hit SKIP in the top right corner. If I hit SKIP, I get this message:

~~~
Something strange occurred
Sorry about that. Please try again.
~~~

If I try again, same error. If I then disable proxy and try again, it works.

**Comment by 89z** [comment]
Update: I was able to fix this using the method described here:

https://github.com/shroudedcode/apk-mitm/issues/34#issuecomment-770393173

However in my case, the file was:

~~~
smali_classes2/g0/r.smali
~~~

and the method was:

~~~
a(Ljava/lang/String;Lkotlin/jvm/functions/Function0;)V
~~~

**Comment by shroudedcode** [comment]
Closing this since the issue should eventually be fixed by #59.
